plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -898,13 +946,18 @@ fu_genesys_usbhub_device_setup(FuDevice *device, GError **error)
 			self->fw_key_addr[1] = 0xE000;
 			self->code_size[0] = self->code_size[1] = self->fw_data_total_count;
 		}
+		self->dynamic_ts.bonding =
+		    _fu_genesys_revert_hex_char_to_int(self->dynamic_ts.bonding);
+		if (self->static_ts.tool_string_version < TOOL_STRING_VERSION_BONDING_QC)
+			self->dynamic_ts.bonding <<= 1;
 		if (self->dynamic_ts.running_mode == 'M') {
 			self->running_bank = BANK_MASK_CODE;
 		} else if ((self->dynamic_ts.bonding & 0x10) > 0) {
 			self->running_bank = BANK_SECOND;
 		} else {
 			self->running_bank = BANK_FIRST;
 		}
+		fu_device_add_instance_u8(device, "BONDING", (self->dynamic_ts.bonding & 0x0F));
```

Comment:
use the local variable

Reply:
Is that possible to have a macro for the 0x0F value, as for GL3590?

Reply:
Add GL3523_BONDING_VALID_BIT for that.